content/docs/details/techdocs/build-via-ci.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Build and Publish TechDocs via CI
+publishedDate: '2023-09-18'
+description: Build TechDocs via CI
+---
+
+You can build and publish tech docs via your own CI pipelines so that the latest version is always ready to view in Roadie and you don't have to wait for a new build when viewing an updated version of your docs.
```

Comment:
```suggestion
You can build and publish TechDocs via your own CI pipelines so that the latest version is always ready to view in Roadie and you don't have to wait for a new build when viewing an updated version of your docs.
```


It's written like this on the bottom file. I don't think we have decided which way to write these yet?

Reply:
Few other instances on this file

Reply:
It should always be TechDocs if we're talking about the feature. Otherwise we should say "technical documentation".